signinapp/model.py

```diff
@@ -140,11 +141,16 @@ class User(UserMixin, db.Model):
     awards = db.relationship("BadgeAward", back_populates="owner", cascade="all, delete-orphan")
     badges = association_proxy("awards", "badge")
 
+    # Guardian specific data
     guardian_user_data = db.relationship(
         "Guardian", back_populates="user", uselist=False)
     guardians = db.relationship(
         "Guardian", secondary=parent_child_association_table, back_populates="students")
 
+    # Student specific data
```

Comment:
Technically a list of guardians should probably be student specific data too?

Reply:
Yup. Moved it.